electrum/daemon.py

```diff
@@ -239,10 +261,21 @@ def __init__(self, daemon, fd):
     async def run(self):
         self.runner = web.AppRunner(self.app)
         await self.runner.setup()
-        site = web.TCPSite(self.runner, self.host, self.port)
+        if self.socktype == 'unix':
+            site = web.UnixSite(self.runner, self.sockpath)
+        elif self.socktype == 'tcp':
+            site = web.TCPSite(self.runner, self.host, self.port)
+        else:
+            raise Exception(f"unknown socktype '{socktype!r}'")
```

Comment:
flake8 test is failing
```
./electrum/daemon.py:269:30: F821 undefined name 'socktype'
            raise Exception(f"unknown socktype '{socktype!r}'")
                             ^
```
```suggestion
            raise Exception(f"unknown socktype '{self.socktype!r}'")
```

same thing a few lines below

Reply:
Thanks, I missed that.